lib/rbs_rails/path_helpers.rb

```diff
@@ -18,6 +18,12 @@ def generate
         interface _RbsRailsPathHelpers
         #{methods.join("\n").indent(2)}
         end
+
+        module ::ActionController
+          class ::ActionController::Base
+            include ::_RbsRailsPathHelpers
+          end
+        end
```

Comment:
Note: As a preparation for https://github.com/pocke/rbs_rails/pull/304, this RBS uses absolute paths.